### DISCUSSION TITLE
Next.js API function is executed in the wrong region

### DISCUSSION THREAD

**Discussion by mayrsascha** [vercel/community#5269] [Feature Question]
### Question

I have a Next.js app deployed (pages dir) and have a couple of API functions in the pages/api/ folder. 

In my project I set the function region to Frankfurt, as that's where my database is hosted as well:

<img width="1017" alt="image" src="https://github.com/vercel/community/assets/29578226/42d9ed3c-14a3-4d45-93ac-5d5cbad44b3d">

But when I check the API function logs, it says the region it gets executed is Singapore:

<img width="1180" alt="image" src="https://github.com/vercel/community/assets/29578226/f0744349-2fd3-4949-9e8d-827034adbc4f">

The API function times out after 10 seconds, locally this execution is much faster...

How can I change this API function to be executed in Frankfurt as it says in my Vercel project settings?

**Comment by alpbozkurt** [comment]
Having the exact same problem.

Were you able to resolve this @mayrsascha ?
